src/lib/utils/helpers.ts

```diff
@@ -284,3 +292,9 @@ export function convertSarcoPerSecondToPerMonth(diggingFeePerSecond: string): st
 }
 
 export const wait = (ms: number) => new Promise(res => setTimeout(res, ms));
+
+export async function getCurrentTimeSec(provider: Provider | Web3Provider) {
```

Comment:
Could add a return type to this function signature

Reply:
Yup, that would be most wise indeed.